README.md

```diff
@@ -7,12 +7,11 @@ Hi! I'm Anthony. I'm a 27 year old web developer based in New York. As of late I
 
 ## Personal Projects
 
- - [Ephemeral Heart](https://www.ephemeralheart.com/)  
+ - [Ephemeral Heart](https://www.ephemeralheart.com/) - A site where anonymous users submit messages which are then read and judged by other anonymous users.
```

Comment:
Might as well just explain the project where it's mentioned, then go into the building blocks.